extensions/youdao-translate/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "https://json.schemastore.org/tsconfig",
   "display": "Node 16",
-  "include": ["src/**/*"],
+  "include": ["src/**/*", "types.ts"],
```

Comment:
I don't think it's necessary. You can put the `types.ts` file into the `src` directory.